packages/pwa-buildpack/src/WebpackTools/PWADevServer.js

```diff
@@ -41,11 +43,18 @@ const PWADevServer = {
         debug('configure() invoked', config);
         PWADevServer.validateConfig('.configure(config)', config);
         const devServerConfig = {
+            public: process.env.PWA_STUDIO_PUBLIC_PATH || '',
             contentBase: false, // UpwardPlugin serves static files
             compress: true,
             hot: true,
+            watchOptions: {
```

Comment:
had to add the `watchOptions` back in for [hot reloading](https://github.com/Atyantik/react-pwa/issues/42) to work properly